owl/EASE-SAY.owl

```diff
@@ -171,6 +404,21 @@ Image schemas are considered as fundamental, pre-conceptual, building blocks of
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#InstrumentRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#InstrumentRole">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#classifies"/>
+                <owl:someValuesFrom rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalArtifact"/>
```

Comment:
a role can exist without ever being manifested, so universal restriction should be used instead